compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Monoid.hs

```diff
@@ -11,11 +11,59 @@ module DA.Daml.Doc.Render.Monoid
 import DA.Daml.Doc.Types
 import Control.Monad
 import Data.Foldable
+import Data.Maybe
+import Data.List.Extra
 import System.FilePath
 import qualified Data.Map.Strict as Map
 import qualified Data.Set as Set
 import qualified Data.Text as T
 
+data RenderOut
+    = RenderSpaced [RenderOut]
+    | RenderModuleHeader T.Text
+    | RenderSectionHeader T.Text
+    | RenderAnchor Anchor
+    | RenderBlock RenderOut
+    | RenderList [RenderOut]
+    | RenderFields [(RenderText, RenderText, RenderText)]
```

Comment:
Might be nice to either rename this to `RenderRecordFields` (I’m happy calling fields in templates and choices record fields :)) or add a comment here. I first got slightly confused as to what `Fields` refers to here.